spectrum-offchain/src/backlog/persistence.rs

```diff
@@ -9,7 +13,51 @@ where
     TOrd: OnChainOrder,
 {
     async fn put(&mut self, ord: BacklogOrder<TOrd>);
-    async fn exists(&self, ord: TOrd) -> bool;
+    async fn exists(&self, ord_id: TOrd::TOrderId) -> bool;
     async fn drop(&mut self, ord_id: TOrd::TOrderId);
     async fn get(&self, ord_id: TOrd::TOrderId) -> Option<BacklogOrder<TOrd>>;
 }
+
+#[async_trait(?Send)]
+impl<TOrd> BacklogStore<TOrd> for RocksDBClient
+where
+    TOrd: OnChainOrder + Serialize + DeserializeOwned + Send + 'static,
+    TOrd::TOrderId: Serialize + DeserializeOwned + Send,
+{
+    async fn put(&mut self, ord: BacklogOrder<TOrd>) {
+        let db = self.db.clone();
+        spawn_blocking(move || {
+            db.put(
+                bincode::serialize(&ord.order.get_self_ref()).unwrap(),
```

Comment:
The key that's used is simply the serialized representation of `TOrderId`. Do you want a particular prefix/postfix for the key?

Reply:
No need for that